packages/generator-single-spa/src/util-module/templates/util-module.package.json

```diff
@@ -1,11 +1,13 @@
 {
+  "name": "<%= name %>",
   "scripts": {
     "start": "webpack serve",
     "start:standalone": "webpack serve --env standalone",
     "build": "webpack --mode=production",
     "analyze": "webpack --mode=production --env analyze",
     "lint": "eslint src --ext js<% if (typescript) { %>,ts,tsx<% } %>",
-    "format": "prettier --write './**'",
+    "format": "prettier --write .",
+    "check-format": "prettier --check .",
```

Comment:
Would these also need the `--plugin-search-dir` configured like it is in other places? Like here: https://github.com/single-spa/create-single-spa/pull/263/files#diff-08b4b9caf9be2ad1db58cabfa38f1c89d699482101603f1c21705a60874c8346R8-R9

Reply:
No my understanding is that `plugin-search-dir` is only needed for prettier-plugin-svelte, see https://github.com/sveltejs/prettier-plugin-svelte#how-to-use-cli

Reply:
Yea, interesting. That shouldn't be necessary since the default Prettier behavior is that

> Plugins are automatically loaded if you have them installed in the same node_modules directory where prettier is located. 
>
> https://prettier.io/docs/en/plugins.html 

So the way its configured in the prettier-plugin-svelte README seems superfluous. Don't think it _needs_ to be changed though.